native/components/message-markdown.react.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import { Component } from 'react';
+import * as React from 'react';
+import { View, ViewPropTypes } from 'react-native';
+import { merge, pick } from 'lodash';
+import PropTypes from 'prop-types';
+import SimpleMarkdown from 'simple-markdown';
+
+import styles from '../themes/markdown-styles';
+import darkStyles from '../themes/markdown-dark-styles';
+import rules from './message-markdown-rules.react';
+
+type Props = {
```

Comment:
We've talked before about avoiding the ambiguous `{ }` syntax. Always prefer exact syntax `{| |}`, or inexact syntax `{...}` if you need it.

Please try to catch these sorts of errors next time before submitting the PR! In my ideal world, I only ever have to give you feedback on something like this once for you to incorporate it into your future work.

A good approach is to always run `git diff master...` before submitting your PR, and imagine that you're me reading it. Try to catch anything I would catch before I do.